boot_default/config.txt

```diff
@@ -73,3 +73,6 @@ dtparam=audio=on
 
 # for more options see http://elinux.org/RPi_config.txt
 
+[pi3]
+# for light board
+enable_uart=1
```

Comment:
I suppose that this comment also applies to https://github.com/KanoComputing/kano-updater/pull/185